charts/sonarqube/values.yaml

```diff
@@ -519,7 +521,10 @@ sonarqubeFolder: /opt/sonarqube
 tests:
   image: ""
   enabled: true
-  resources: {}
+  resources:
+    limits:
+      cpu: 800m
+      memory: 6144M
```

Comment:
For the test pods i think we can have request/limits much lower, as we only execute kind of a bash script that ping the APP-UI. This will not trigger the JVM creation process if I recall correctly, we overwrite the entrypoint.

Reply:
yes, we just do a wget or something similar.  This is a fair point because if we don't specify the requests, the limits we set will become requests too and this is not what I wanted...lowering the limits now :)